python/setup.py

```diff
@@ -354,6 +468,19 @@ def append_cmake_bool(value, varname):
                 shutil.move(pjoin(build_prefix, 'include'),
                             pjoin(build_lib, 'pyarrow'))
 
+                # We need to, again, add the PyArrow cpp include folder
+                build_pyarrow_cpp_include = pjoin(
+                    saved_cwd, 'build/dist/include')
+                if not os.path.isdir(
+                        pjoin(build_pyarrow_cpp_include, "arrow", "python")):
+                    self.mkpath(
```

Comment:
We are making the source path we want to copy? (if we need to create something, I would expect the target directory)

Reply:
Hope I understood correctly, the changes I made (together with updated comment from https://github.com/apache/arrow/pull/13311#discussion_r926243308):

```python
```